src/Shared/Recruit.Vacancies.Client/Application/Validation/Fluent/FluentVacancyValidator.cs

```diff
@@ -479,6 +481,29 @@ private void ValidateWorkingWeek()
             });
         }
 
+        private void ValidateTraineeshipWorkingWeek()
+        {
+            When(x => x.Wage != null, () =>
+            {
+                RuleFor(x => x.Wage.WorkingWeekDescription)
+                    .Cascade(CascadeMode.StopOnFirstFailure)
+                    .NotEmpty()
+                        .WithMessage("Enter weekly hours on the traineeship")
+                        .WithErrorCode("37")
+                    .ValidFreeTextCharacters()
+                        .WithMessage("Weekly hours details contain some invalid characters")
+                        .WithErrorCode("38")
+                    .MaximumLength(250)
+                        .WithMessage("Weekly hours on the traineeship must not exceed {MaxLength} characters")
+                        .WithErrorCode("39")
+                    .ProfanityCheck(_profanityListProvider)
+                    .WithMessage("Weekly hours details must not contain a banned word or phrase.")
```

Comment:
The described field is inconsistent, might need some content input but feel it should be `Weekly hours on the traineeship contain some invalid characters` and `Weekly hours on the traineeship must not contain a banned word or phrase`